README.md

```diff
@@ -15,114 +15,441 @@
 
 </div>
 
-This project provides a basic method to manage system configuration using [Nix][nixos]
-on any Linux distribution.
-It builds on the many [modules that already exist in NixOS](https://github.com/NixOS/nixpkgs/tree/master/nixos/modules).
+System Manager brings the power of NixOS-style declarative configuration to other Linux distributions. Describe what your system should look like, by specifying packages, services, and settings all in Nix, then apply it safely and atomically with a single command. Each change is reproducible and soon will be rollback-ready, just like NixOS generations.
 
-*Warning*: System Manager is a work in progress, you can expect things not to work or to break.
+If you're familiar with Home Manager, think of it as being similar, but for your entire machine. Whereas Home Manager manages user environments, System Manager manages the entire system, starting at root-level configurations, packages, and services, using the same reliable, Nix-based model.
 
-## Usage
+System Manager builds on the many modules that already exist in [NixOS].
 
-### Install Nix
+## Quick Example to Get Started
 
-In order to use System Manager, you will first need to install Nix.
-You can either use your distro's package manager, or use one of the available options
-to install Nix.
+We will assume you're using a non-NixOS distrubution (such as Ubuntu) and you have Nix already installed.
```

Comment:
```suggestion
We will assume you're using a non-NixOS distribution (such as Ubuntu) and you have Nix already installed, with flakes enabled.
```

Not a blocker but if we can assume that flakes are enabled, we can remove a bunch of edge-cases below.

Let's say we have a separate page that talks about installing Nix with flakes enabled, then we can keep linking back to it.